python/ray/tune/ray_trial_executor.py

```diff
@@ -155,10 +152,10 @@ def pause_trial(self, trial):
         before pausing, which is restored when Trial is resumed.
         """
 
+        super(RayTrialExecutor, self).pause_trial(trial)
```

Comment:
Should update the function doc here.

Reply:
Turns out this would remove the trial_future, so I rearranged this again (and is again consistent with the doc)